challenge.js

```diff
@@ -9,6 +9,7 @@
  */
 
 // ANSWER
+call document.create.Element
```

Comment:
Yeah, that will create a new HTML element object. However, it will not attach it to the DOM; it will be a floating, fully-functional HTML element that is only accessible via JavaScript. 1